drivers/ng_at86rf2xx/ng_at86rf2xx_netdev.c

```diff
@@ -293,7 +293,8 @@ static void _receive_data(ng_at86rf2xx_t *dev)
     /* fill missing fields in netif header */
     netif = (ng_netif_hdr_t *)hdr->data;
     netif->if_pid = dev->mac_pid;
-    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len);
+    /* LQI offset: Payload (pkt_len) + FCS (2 byte) */
+    ng_at86rf2xx_rx_read(dev, &(netif->lqi), 1, pkt_len + 2);
```

Comment:
Looking at section 38.3.2 of the samr21 reference it seems there has to be one additional offset byte as PHR has offset 0. Right? 
<img width="649" alt="screen shot 2015-07-06 at 11 10 40" src="https://cloud.githubusercontent.com/assets/56618/8519062/f54512c4-23cf-11e5-8beb-53779d0dad21.png">

While the offset should be fine for rf231 modules.